dataland-frontend/tsconfig.cypress-ct.json

```diff
@@ -7,7 +7,8 @@
     "build/clients/**/*",
     "tests/component/**/*",
     "tests/sharedUtils/**/*",
-    "./custom-adjustments-for-cypress-ct-tsconfig.d.ts"
+    "./custom-adjustments-for-cypress-ct-tsconfig.d.ts",
+    "src/assets/content.json"
```

Comment:
Why? Let's have a call.

Reply:
Let's also talk about this in our call.

Reply:
Clarified in a test that Florian will try removing this.

Reply:
We do need that include.